### PR TITLE
Update QAT and BN listing with additions/removals

### PR DESCRIPTION
**Pull request by Xinnoh**
There have been some additions to the qat who will not be focused on a specific mode, they are focusing only on moderation or metadata, hence they are listed without a mode.

### CODE REVIEW

**Comment by Ephemeralis** [comment]
confusing as shit to have a table that contains categorical designation and then have no categorical designation for the new QAT

some caveat about "this is a metadata qat" needs to be added or the table needs to have a new column added or something, can't just have an empty row

File: wiki/People/Quality_Assurance_Team/en.md [hunk 1]
```diff
@@ -15,21 +15,25 @@ Team Members
...
-| [Nardoxyribonucleic](https://osu.ppy.sh/users/876419) | ![No][false] | ![Yes][true] | ![No][false] | ![No][false] | Chinese, Cantonese         |
-| [p3n](https://osu.ppy.sh/users/123703)                | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | German                     |
-| [pishifat](https://osu.ppy.sh/users/3178418)          | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] |                            |
-| [Protastic101](https://osu.ppy.sh/users/6712747)      | ![No][false] | ![No][false] | ![No][false] | ![Yes][true] |                            |
-| [Raiden](https://osu.ppy.sh/users/2239480)            | ![No][false] | ![Yes][true] | ![No][false] | ![No][false] | Spanish, Catalan           |
+| Name                                              | osu!         | osu!taiko    | osu!catch    | osu!mania    | Additional Languages       | Primary Branch       |
+|---------------------------------------------------|:------------:|:------------:|:------------:|:------------:|----------------------------:|----------------------------:|
```
Comment (MegaApplePi) [inline wiki/People/Quality_Assurance_Team/en.md]: Why use right align for the last two columns?

Comment (Xinnoh) [reply wiki/People/Quality_Assurance_Team/en.md]: that's' how it was like, but moved left

**Comment by Xinnoh** [comment]
discussed with qat, removing the branches thing for now since they'll be doing their own thing

**Comment by Xinnoh** [comment]
more discussion, reformatted the nominator list to be sorted by modes

File: wiki/People/Beatmap_Nominators/en.md [hunk 2]
```diff
@@ -43,119 +41,147 @@ Modders that aim to apply to the Beatmap Nominators must fulfill the following c
...
-| [Wafu](https://osu.ppy.sh/users/888955)               | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | Czech                    |
-| [Weber](https://osu.ppy.sh/users/6410432)             | ![No][false] | ![No][false] | ![No][false] | ![Yes][true] |                          |
-| [Wishkey](https://osu.ppy.sh/users/1101704)           | ![Yes][true] | ![No][false] | ![No][false] | ![No][false] | Dutch, German            |
-| [xfraczynho](https://osu.ppy.sh/users/2192669)        | ![No][false] | ![Yes][true] | ![No][false] | ![No][false] | Spanish                  |
-| [YaHao](https://osu.ppy.sh/users/2633753)             | ![No][false] | ![No][false] | ![No][false] | ![Yes][true] | Chinese                  |
-| [ZiRoX](https://osu.ppy.sh/users/200768)              | ![No][false] | ![No][false] | ![Yes][true] | ![No][false] | Spanish                  |
+The tables listed below indicate the game mode(s) in which each Beatmap Nominator is formally qualified to nominate.
+_Please note: All BN speak English unless otherwise noted._ Link to the [user group page](https://osu.ppy.sh/groups/28).
```
Comment (MegaApplePi) [inline wiki/People/Beatmap_Nominators/en.md]: prepend an empty line or merge this line with 61

File: wiki/People/Beatmap_Nominators/en.md [hunk 3]
```diff
@@ -43,119 +41,147 @@ Modders that aim to apply to the Beatmap Nominators must fulfill the following c
...
+| [Kawawa](https://osu.ppy.sh/users/4647754)            | Korean                   |
+| [Maxus](https://osu.ppy.sh/users/4335785)             | Indonesian               |
+| [Rivals_7](https://osu.ppy.sh/users/4610379)          | Indonesian               |
+| [Sandalphon](https://osu.ppy.sh/users/2633753)        | Chinese                  |
+| [Tofu1222](https://osu.ppy.sh/users/6089608)          | Chinese                  |
+| [Weber](https://osu.ppy.sh/users/6410432)             |                          |
+
+
+
+
```
Comment (MegaApplePi) [inline wiki/People/Beatmap_Nominators/en.md]: :eyes: empty lines

File: wiki/People/Beatmap_Nominators/en.md (hunk 2 above)
Comment (Xinnoh) [reply wiki/People/Beatmap_Nominators/en.md]: done

File: wiki/People/Beatmap_Nominators/en.md (hunk 3 above)
Comment (Xinnoh) [reply wiki/People/Beatmap_Nominators/en.md]: 👀

**Pull request merged**